src/components/sidenav/drawer.html

```diff
@@ -0,0 +1 @@
+<ng-content></ng-content>
```

Comment:
Can you rename the files that start with `drawer` to reflect the public api name?